pkg/cmd/admin/groups/sync/rfc2307/ldapinterface.go

```diff
@@ -57,6 +59,9 @@ type LDAPInterface struct {
 	// cachedUsers holds the result of user queries for later reference, indexed on user UID
 	// e.g. this will map an LDAP user UID to the LDAP entry returned from the query made using it
 	cachedUsers map[string]*ldap.Entry
+
+	// Err is used to log errors found interfacing with LDAP
+	Err io.Writer
```

Comment:
> after consideration, I've decided @deads2k can stay on as the "take vague statements from @liggitt and turn them into interface definitions" person

We should create an @deadliggitt account and make it a game to see if they can tell who's making the comments that day. :)